libs/auth/src/lib/services/auth.interceptor.ts

```diff
@@ -22,11 +24,16 @@ export class AuthInterceptor implements HttpInterceptor {
                 redirect_uri: window.location.origin,
                 response_type: this._oauthService.responseType,
             });
-            return throwError(new HttpErrorResponse({ status: 401, statusText: 'Unauthorized' }));
-        } else if (!token) {
-            return throwError(new HttpErrorResponse({ status: 401, statusText: 'No Token' }));
+            return EMPTY;
+        }  else if (!token) {
+          return EMPTY;
```

Comment:
Is there no logout required when no token is present? Does the redirect to the login form still happen?

Reply:
I remembered the logic we were using there: 
If I have a token in the storage but it is invalid, I shoud be logged out
If I dont have a token, then the login process has not yet finished (oauth2 flow)
If i have a token and it is valid (the else-case), then we redirect to error page when an error occurs

So if I added the logout in the !token case, then there would be an indefinite redirect to the login page.

Reply:
Alright :)